borg/helpers.py

```diff
@@ -519,6 +519,30 @@ def dir_is_tagged(path, exclude_caches, exclude_if_present):
     return tag_paths
 
 
+def format_line(formatstr, data):
+    """Filter out unwanted properties of str.format(), because "formatstr"
+    is user provided.
+    """
```

Comment:
I wanted to leave clear place where to do filtering if needed. At this point I could not think any such filter. 
So kinda superfluous at the moment.